pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -387,13 +389,13 @@ benchmarks! {
 		)?;
 	} verify {
 		assert!(
-			Pallet::<T>::candidate_state(&caller).unwrap().request.is_none()
+			Pallet::<T>::candidate_info(&caller).unwrap().request.is_none()
 		);
 	}
 
 	delegate {
```

Comment:
Per our conversation this morning:

`delegate()` is now either cheap (doesn't touch bottom delegations) or expensive (it does). We should do some measurement of this, but assuming it's a drastic difference, we have a few options:

1. Make two extrinsics (probably a bad option)
2. Add a hint like we have elsewhere. I'm not quite sure that this will work well with benchmarking, however. We may at least need to create multiple benchmarks for this to work.
3. Assume worst-case for weight charging and refund. This is unprecedented in our codebase, so it might be a worthwhile experiment in any case.